假人问题修复/优化

LogSpiral opened the pull request:
假人现在默认血量上限为二十亿，可以设置的最高血量为int.maxValue
血量的ui设置现在移除了滑块(范围太大，滑块已经没有意义了)
现在可以设置原版的AIStyle了，并且对于常用的几个(如史莱姆)会自动变更相应别的设置选项，如史莱姆时会受重力且有物块判定
现在打开面板的按键从鼠标滚轮改为了"特殊物品快捷键"，也就是可以自己设置而不是固定滚轮了(默认还是滚轮)
现在可以选择自定义AIStyle，此时会有文本框来输入数字，也有按钮打开wiki的相应页面
![QQ截图20250220161315](https://github.com/user-attachments/assets/a837320a-42ef-4d7d-a05c-54491ceb4ffa)

LogSpiral committed:
添加了几个可选AIStyle
略微美化了一下UI界面

LogSpiral committed:
Merge branch '1.4.4' into dummy-rework

LogSpiral committed:
现在开启锁血的假人不会被击败，优化了修改ai模式时的其它配置变动(如史莱姆模式下自动变为不可物块碰撞+有重力)，修改了immortal的描述，现在可以设置更多AIStyle

Cyrillya commented:
wiki页面的按钮还得再调调，调成自动排列，不过这个无关紧要，不调也行）
英文的“Self Define”建议改成“Custom”
主要是多人模式下自定义AIStyle的问题，直接成为炸服神器了

修改意见：
- 添加一个config选项“多人模式下是否允许假人自定义AIStyle”，默认关闭
- 如果开启了选项，那就跟现在一样随便调
- 如果没开启，那么多人模式下选中“自定义AIStyle”之后，弹出Notification提示“该服务器禁止自定义AIStyle”，然后自动重设回原来的“无AI”。就算玩家以某种方式调到了AIStyle，生成的时候会进一步验证Config，还是强制重设成无AI
- 如果可以的话，加上：点击PopupNotification会自动跳转到【配置中心】的【全部】分类，然后搜索框自动输入这个Config选项的Label，以显示此选项。这样可能会少点疑问？

LogSpiral committed:
Merge branch '1.4.4' into dummy-rework

LogSpiral committed:
优化了开启AIStyle的WIki页面的按钮位置，现在需要开启相应服务器设置才能用自定义的AIStyle

LogSpiral committed:
修改了配置项内容，现在单人模式是可以直接自定义AIStyle的，毕竟主要是防止有人用来炸服务器

Cyrillya committed:
更改自定义AI逻辑与细节调整
- 现在是自定义要模仿的NPC而不是更改aiStyle
- 可以调整Scale了
- UI开启和关闭更流畅了
- 添加了小地图图标
- UI元素风格和LongSwitch保持一致
- 部分游戏专有名词英文本地化修复
- 修复DropdownList不置顶问题

Cyrillya committed:
假人的最终细节调整
